src/officehours_api/backends/zoom.py

```diff
@@ -122,10 +122,12 @@ def _get_access_token(cls, user: User) -> str:
                 },
                 headers=cls._get_client_auth_headers(),
             )
-            if resp.status_code == 401:
+            if resp.status_code >= 400 and resp.status_code < 500:
                 # The refresh_token was invalidated somehow.
                 # Maybe the user removed our Zoom app's auth.
                 # Force them to be prompted again.
+                # The specific code sent by Zoom has changed before,
+                # so it now checks for any client error during refresh.
                 zoom_meta = {}
```

Comment:
This looks fine. I might log a info message when this happens with the `resp.status_code` and the `user.get_username().` just incase there are unexpected errors that come up , but this might be hard to test so I'm fine with it as-is.

Reply:
Thanks! I was just working on this. I added a log message here (https://github.com/tl-its-umich-edu/remote-office-hours-queue/pull/364/commits/99ed7cc5b9508bccbab3a72fb4bbadecf159a07f) that includes the user ID (to keep the username itself out of the log) so we know when the reset has actually happened. I was discussing with @zqian this morning, the way the code is written, an exception will still be raised, so we'll see in the logs and in email what the status code was from Zoom.

Reply:
Okay that makes sense that it still hits the `raise_for_status`. We just didn't know the id of the user affected.

This is similar to Student Explorer where the user ID is logged when an exception occurs but even there there's a request to add it to the email too. I think for this app having it in the logs is fine as we want to reduce review and reduce the email frequency.

Reply:
Yeah, handling exception and broken URL emails will have to be a separate issue, hopefully we can do that at some point.